src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -89,5 +89,12 @@ export default {
 			loanId: Number(this.$route.params.id || 0),
 		};
 	},
+	created() {
```

Comment:
I think this needs to move the `mounted` hook inside the SummaryCard.vue component.

The `status` property is populated by that query and calling in mounted ensure it's set before firing the event.